apps/charterafrica/src/theme/index.js

```diff
@@ -82,6 +82,12 @@ const theme = createTheme({
     h4Small: initializeTypographyVariant(23, 28, 700, PROMPT_FONT_FAMILY),
     h5: initializeTypographyVariant(23, 28, 700, PROMPT_FONT_FAMILY),
     h5Small: initializeTypographyVariant(19, 23, 700, PROMPT_FONT_FAMILY),
+    h5SmallSemiBold: initializeTypographyVariant(
```

Comment:
Make sure this new variant has component mapping [here](https://github.com/CodeForAfrica/ui/pull/317/files#diff-418befd186fb8ea0a66ca84303e3a9992c24baccd2133765e162281958eb296cL124-L125).